bokeh/core/json_encoder.py

```diff
@@ -49,7 +49,7 @@
 pd = import_optional('pandas')
 rd = import_optional("dateutil.relativedelta")
 
-NP_EPOCH = np.datetime64('1970-01-01T00:00:00Z')
+NP_EPOCH = np.datetime64(0, 's')
```

Comment:
Note that you probably want to set the appropriate time units here for the constant, though I think this gets cast properly in arithmetic. NumPy's default time resolution is `us`.